src/utils/request-router.ts

```diff
@@ -43,6 +43,10 @@ export class RequestRouter {
         return this._regionCache[this.apiHost]
     }
 
+    isUsingProxy(): boolean {
+        return this.instance.config.api_host != null && this.instance.config.api_host !== defaultConfig().api_host
```

Comment:
```suggestion
        return !_isNullish(this.instance.config.api_host) && this.instance.config.api_host !== defaultConfig().api_host
```

doesn't make much difference at each call site but when we replaced all the `=== undefined` calls it was 1% of the bundle size in saving 🤯 

in terms of implementation @benjackwhite is way more familiar with all of this than me right now but I think this is correct (except for self hosted maybe)

Reply:
Wow interested, good to know RE: _isNullish

Reply:
This doesn't work :) The rest of the code in `request-router` has the context as to why not.

The given `api_host` is almost always going to be different to the default as we now have 2 regions (so no real default anymore). 

We will likely change a lot of this over time. Instead this is already solved by the `region` property. You can remove this method and instead just check `this.requestRouter.region === RequestRouterRegion.CUSTOM`.

Reply:
I was thinking that was the case, appreciate the confirmation.